README.md

```diff
@@ -168,6 +168,19 @@ example.org {
 }
 ~~~
 
+IP addresses are also allowed. They are automatically converted to reverse zones:
+
+~~~ txt
+10.0.0.0/24 {
+    # ...
+}
+~~~
+Means you are authoritative for `0.0.10.in-addr.arpa.`. 
+
+The netmask must be dividable by 8, if it is not the reverse conversion is not done. This also works
```

Comment:
Shouldn't we implement https://www.ietf.org/rfc/rfc2317.txt for classless networks?